### PR TITLE
DM-27414: Rename Hexapod to MTHexapod

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: doc/version_history.rst [hunk 1]
```diff
@@ -1,11 +1,31 @@
...
+v0.10.0
+=======
+
+Changes:
+
+* Updated to use and require ts_xml 7, ts_idl 2.2, and ts_hexrotcomm 0.11:
+
+    * Rename SAL component and ts_idl enum module ``Hexapod`` to ``MTHexapod``.
+
+* Renamed the package to ``ts_hexapod``.
```
Comment (wvreeven) [inline doc/version_history.rst:18]: Shouldn't that be ts_mthexapod?

File: setup.py [hunk 2]
```diff
@@ -16,7 +16,7 @@
 data_files_path = tools_path.relative_to(base_prefix).parents[1]
 
 setuptools.setup(
-    name="ts_hexapod",
+    name="ts_mthexapod",
     description="CSC for main telescope hexapods (camera and secondary mirror)",
     use_scm_version={
         "write_to": "python/lsst/ts/hexapod/version.py",
```
Comment (wvreeven) [inline setup.py]: Shouldn't that be python/lsst/ts/mthexapod/version.py ?

File: setup.py [hunk 3]
```diff
@@ -28,12 +28,12 @@
     packages=setuptools.find_namespace_packages(where="python"),
     package_data={"": ["*.rst", "*.yaml"]},
     scripts=["bin/run_hexapod.py", "bin/command_hexapod.py"],
```
Comment (wvreeven) [inline setup.py:30]: These scripts were renamed.

File: tests/test_csc.py [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# This file is part of ts_hexapod.
+# This file is part of ts_mthexapod.
 #
 # Developed for the LSST Data Management System.
```
Comment (wvreeven) [inline tests/test_csc.py:3]: Copyright statements referring to the LSST Data Management System should be updated to refer to the LSST Telescope and Site Systems but that can be done at a later time.

**Review by wvreeven — APPROVED** [review]
There are a few mentions of hexapod that slipped your attention and otherwise this looks good to me.

File: setup.py (hunk 2 above)
Comment (r-owen) [reply setup.py]: Good catch. And I may have missed the change to the bin scripts a few lines later. In any case it's all fixed now.

File: tests/test_csc.py (hunk 4 above)
Comment (r-owen) [reply tests/test_csc.py:3]: Interesting. Yes. I'll fix it now.

**Comment by r-owen** [comment]
Thank you very much for catching those issues.

I did a global search for _hexapod and found several instances that needed changing to _mthexapod (such as the bin scripts in setup.cfg). I also fixed the version.py in setup.py.

File: tests/test_csc.py (hunk 4 above)
Comment (r-owen) [reply tests/test_csc.py:3]: If I can figure out what to fix it to. I'm pretty sure LSST is not the right thing. I've asked on the ts software Slack channel, but not response yet.

Comment (r-owen) [reply tests/test_csc.py:3]: At this point, getting no answer, I think I'll hold off for another ticket, as you suggested.

**Pull request merged**